shared/js/smart-screen/f_t_e_wizard.js

```diff
@@ -0,0 +1,192 @@
+/* global SimpleKeyNavigation, KeyEvent */
```

Comment:
According to Fernando, it would be good to add MPL License in new files.

```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```